.travis.mk

```diff
@@ -89,7 +89,7 @@ deps_debian:
 		libcurl4-openssl-dev libunwind-dev libicu-dev \
 		python python-pip python-setuptools python-dev \
 		python-msgpack python-yaml python-argparse python-six python-gevent \
-		lcov ruby clang llvm llvm-dev zlib1g-dev autoconf automake libtool
+		lcov ruby clang llvm llvm-dev zlib1g-dev
```

Comment:
The comment above states 'It is highly recommended to only add dependencies (don't remove them)'. Is there a reason to overcome the rule? If so, please, explicitly mention it in the commit message.